kolibri/core/assets/src/views/app-body/index.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+
+  <!-- class unused, used as identifier when debugging from DOM -->
+  <div class="app-body" :style="contentStyle">
+    <loading-spinner v-if="loading" />
+    <template v-else>
+      <error-box v-if="error" />
+      <slot></slot>
+    </template>
+  </div>
+
+</template>
+
+
+<script>
+
+  import loadingSpinner from 'kolibri.coreVue.components.loadingSpinner';
+  import errorBox from '../error-box';
+
+  export default {
+    name: 'appBody',
+    components: {
+      loadingSpinner,
+      errorBox,
+    },
+    props: {
+      padding: {
+        type: Number,
+        required: true,
+      },
+      // reserve space at the top for appbar
+      topGap: {
+        type: Number,
+        required: true,
+        default: 0,
+      },
+      // reserve space at the bottom for floating widgets
+      bottomGap: {
+        type: Number,
+        required: false,
+        default: 0,
+      },
+    },
+    computed: {
+      contentStyle() {
+        return {
+          top: `${this.topGap}px`,
+          bottom: `${this.bottomGap}px`,
+          padding: `${this.padding}px`,
+        };
+      },
+    },
+    watch: {
+      documentTitle: 'updateDocumentTitle',
```

Comment:
I would have expected the document title to remain the responsibility of `core-base`. Am I misunderstanding the intent?

Reply:
Indeed, now that the `core-base` is indeed the base component for everything again, it would make more sense for this to be back there.